scripts/bootstrap-logkeeper.sh

```diff
@@ -49,4 +49,4 @@ fi
 
 
 echo "Use the following command to start logkeeper:"
```

Comment:
it could be helpful to mention that the command should be run from your logkeeper directory (wherever it exists)